geniza/settings/local_settings.py.sample

```diff
@@ -7,6 +7,7 @@ DEBUG = True
 
 FEATURE_FLAGS = [
```

Comment:
fwiw, there's a [django flags](https://github.com/cfpb/django-flags) library for toggling feature flags via the db so this can be controlled by admin users.

I've never used it myself and not sure if it's totally compatible with our usage here (for the warning banner at least), but thought I'd mention in case it's relevant at some point and you're not aware of it.